rest-api-spec/src/main/resources/rest-api-spec/api/indices.put_mapping.json

```diff
@@ -19,155 +19,10 @@
-              "deprecated":true
-            }
-          },
-          "deprecated":{
-            "version":"7.0.0",
-            "description":"Specifying types in urls has been deprecated"
-          }
-        },
-        {
-          "path":"/{index}/_mapping/{type}",
-          "methods":[
-            "PUT",
-            "POST"
-          ],
-          "parts":{
-            "index":{
-              "type":"list",
-              "description":"A comma-separated list of index names the mapping should be added to (supports wildcards); use `_all` or omit to add the mapping on all indices."
-            },
-            "type":{
-              "type":"string",
-              "description":"The name of the document type",
-              "deprecated":true
-            }
-          },
-          "deprecated":{
-            "version":"7.0.0",
-            "description":"Specifying types in urls has been deprecated"
-          }
-        },
-        {
-          "path":"/{index}/{type}/_mappings",
-          "methods":[
-            "PUT",
-            "POST"
-          ],
-          "parts":{
-            "index":{
-              "type":"list",
-              "description":"A comma-separated list of index names the mapping should be added to (supports wildcards); use `_all` or omit to add the mapping on all indices."
-            },
-            "type":{
-              "type":"string",
-              "description":"The name of the document type",
-              "deprecated":true
-            }
-          },
-          "deprecated":{
-            "version":"7.0.0",
-            "description":"Specifying types in urls has been deprecated"
-          }
-        },
-        {
-          "path":"/{index}/_mappings/{type}",
-          "methods":[
-            "PUT",
-            "POST"
-          ],
-          "parts":{
-            "index":{
-              "type":"list",
-              "description":"A comma-separated list of index names the mapping should be added to (supports wildcards); use `_all` or omit to add the mapping on all indices."
-            },
-            "type":{
-              "type":"string",
-              "description":"The name of the document type",
-              "deprecated":true
-            }
-          },
-          "deprecated":{
-            "version":"7.0.0",
-            "description":"Specifying types in urls has been deprecated"
-          }
-        },
-        {
-          "path":"/_mappings/{type}",
-          "methods":[
-            "PUT",
-            "POST"
-          ],
-          "parts":{
-            "type":{
-              "type":"string",
-              "description":"The name of the document type",
-              "deprecated":true
-            }
-          },
-          "deprecated":{
-            "version":"7.0.0",
-            "description":"Specifying types in urls has been deprecated"
-          }
-        },
-        {
-          "path":"/{index}/_mappings",
```

Comment:
Curious why this test case have been removed? It is legitimate `/{index}/_mappings`, right?

Reply:
Hi @reta ,
Thanks, for pointing out. You are right, this is a legitimate api spec. Let me add this back in a follow up PR.

Reply:
Hi @reta,

Sorry for delay on my end to this. I just noticed that `_mappings` specific spec is marked deprecated as pasted below. Looks like _mappings is only accepted but not documented. Please let me know your thoughts on this. 

```
"deprecated":{
            "version":"7.0.0",
            "description":"The plural mappings is accepted but only /_mapping is documented"
          }
```

Reply:
Ah .. I see, missed that, thanks @dreamer-89 !